<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1048</width>
    <height>561</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>1048</width>
    <height>561</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Panel de Director</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>icons/app.png</normaloff>icons/app.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="menu_desplegable" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>200</width>
      <height>751</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>200</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget{
	background-color: rgb(230, 230, 230);
	color: black;
}

QPushButton{
	height: 30px;
	border:none;
}</string>
    </property>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>50</width>
       <height>50</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>icons/app.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>40</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>CheckSalud</string>
     </property>
    </widget>
    <widget class="QPushButton" name="medicos">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>190</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton:checked{
	background-color: rgb(255, 255, 255);
	border-radius:3px;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>icons/medics.png</normaloff>
       <normalon>icons/on_medics.png</normalon>icons/medics.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>300</width>
       <height>40</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
     <property name="autoExclusive">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="informacion">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>390</y>
       <width>151</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton:checked{
	background-color: rgb(255, 255, 255);
	border-radius:3px;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>icons/icon_informacion.png</normaloff>
       <normalon>icons/on_informacion.png</normalon>icons/icon_informacion.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>300</width>
       <height>40</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="autoExclusive">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="salir">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>460</y>
       <width>151</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>icons/icon_salir.png</normaloff>
       <normalon>icons/on_salir.png</normalon>icons/icon_salir.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>300</width>
       <height>40</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="autoExclusive">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="main" native="true">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>0</y>
      <width>1000</width>
      <height>571</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(200, 200, 200);</string>
    </property>
    <widget class="QPushButton" name="btnExportarMedicoPdf">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>100</y>
       <width>121</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>-1</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	
	
	background-color: rgb(157, 0, 0);
	color: white;
	border: none;
	border-radius: 15px;
	font-weight: bold;
	font-size: 12px;
}

QPushButton:hover{
	
	background-color: rgb(13, 35, 41);
}</string>
     </property>
     <property name="text">
      <string>Exportar a PDF</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtBuscarMedico">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>150</y>
       <width>391</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit{
	padding-left:20px;
	border: 1px solid gray;
	border-radius: 10px;
	background-color: white;
	color: black;
}</string>
     </property>
     <property name="placeholderText">
      <string>Buscar médico</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnExportarMedicoExcel">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>100</y>
       <width>121</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>-1</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	
	background-color: rgb(0, 170, 0);
	color: white;
	border: none;
	border-radius: 15px;
	font-weight: bold;
	font-size: 12px;
}

QPushButton:hover{
	
	background-color: rgb(13, 35, 41);
}</string>
     </property>
     <property name="text">
      <string>Exportar a Excel</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="tabla_medicos">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>180</y>
       <width>701</width>
       <height>351</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
QHeaderView::section{
	font-weight: bold;
	background-color: black;
	color:white;
	padding-left: 5px;
    border: 1px;
    border-radius: 3px;
	
}

QTableWidget{ 
	alternate-background-color: #B0EDFB;
	background-color: #F4F9FA;
	border: 1px;
	border-radius: 10px;
}</string>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <column>
      <property name="text">
       <string>Rut</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Nombre</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Apellidos</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Teléfono</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Cesfam</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Usuario</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Estado</string>
      </property>
     </column>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>50</y>
       <width>201</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Listado de médicos</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cbCesfamMedicoBuscar">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>141</width>
       <height>24</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <item>
      <property name="text">
       <string>Seleccionar Cesfam</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Santa Julia</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Padre Hurtado</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Felix de Amesti</string>
      </property>
     </item>
    </widget>
    <widget class="QPushButton" name="btnIngresarMedico">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>100</y>
       <width>121</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>-1</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	
	
	background-color: rgb(56, 111, 167);
	color: white;
	border: none;
	border-radius: 15px;
	font-weight: bold;
	font-size: 12px;
}

QPushButton:hover{
	
	background-color: rgb(13, 35, 41);
}</string>
     </property>
     <property name="text">
      <string>Ingresar</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>salir</sender>
   <signal>toggled(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>253</x>
     <y>481</y>
    </hint>
    <hint type="destinationlabel">
     <x>270</x>
     <y>564</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
